services/udp/service.go

```diff
@@ -137,8 +137,8 @@ func (s *Service) serve() {
 	defer s.wg.Done()
 
 	s.batcher.Start()
+	buf := make([]byte, UDPBufferSize)
```

Comment:
The reason I asked is we tried this change before and it result in corrupted buffers. We meant to look into why, and forgot. It does seem like it a valid change, but it didn't work for us.

Can you revert this change? I really like your client changes, and we want those. We can come back to this issue.